src/app/services/change-email.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+import { AppRoutes } from '../app.routes';
+import { Router } from '@angular/router';
+import { User } from '../classes/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ChangeEmailService {
+
+  constructor(private http: HttpClient, private router: Router) {}
+
+  getCurrentUser(): Observable<any> {
```

Comment:
This probably should be in a different service

Reply:
I put it in the user service.